scala.md

```diff
@@ -1,9 +1,7 @@
 # Scala
 
- * Always specify an explicit sbt.version
- * Ensure you run in production with GC logging enabled
```

Comment:
I assume we are not putting enough new services into production to warrant this being so prominent.  Now we should probably add more recommendations about maintainability e.g. ScalaStyle and library use.  Perhaps one for the scala guild to consider.

Reply:
GC logging without some idea of what you're going to do with it doesn't seem that useful.